Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -5222,6 +5253,18 @@ class X86Assembler {
         m_formatter.immediate8(static_cast<uint8_t>(condition));
     }
 
+    void vcmpps_mrr(PackedCompareCondition condition, int offset, RegisterID base, XMMRegisterID xmm2, XMMRegisterID xmm1)
+    {
+        // https://www.felixcloutier.com/x86/cmpps
+        // VEX.128.0F.WIG C2 /r ib VCMPPS xmm1, xmm2, xmm3/m128, imm8
+        // B    NA    ModRM:reg (w)    VEX.vvvv (r)    ModRM:r/m (r)    Imm8
+        if (condition == PackedCompareCondition::Equal || condition == PackedCompareCondition::NotEqual)
+            m_formatter.vexNdsLigWigCommutativeTwoByteOp(PRE_SSE_00, OP2_CMPPS_VpsWpsIb, (RegisterID)xmm1, (RegisterID)xmm2, (RegisterID)base, offset);
+        else
```

Comment:
So, remove this part. Commutative instructions cannot take memory. Always use `vexNdsLigWigTwoByteOp`